kvrocks.conf

```diff
@@ -96,6 +96,15 @@ dir /tmp/kvrocks
 # We also can send logs to stdout/stderr is as simple as:
 #
 log-dir stdout
+# log-dir log
+
+# To enable the log cleaner. if set yes , glog will check if there are
+# overdue logs whenever a flush is performed
+enablelogcleaner yes
+
+# INFO level log file whose last modified time is greater than logcleanerday
+# will be unlinked
+logcleanerday 0
```

Comment:
Could you please also mention in the comment what is the default value for `logcleanerday` and set it instead of 0? By the way, what will happen if the value of this setting is `0`?

Reply:
thanks , i will add